src/Actions/TimeoutTransitionAction.php

```diff
@@ -37,14 +51,21 @@ public function getCMSFields()
             $transitions[$t->ID] = $t->Title.' ('.$t->NextAction->Title.')';
         }
 
-        $fields->insertBefore('Property',
-            FieldGroup::create('Timeout', [
-                new NumericField('TimeoutCount', 'Wait for this long:'),
-                new DropdownField('TimeoutIncrement', '', $increments),
-                new LiteralField('', '<span style="font-size:30px;margin-right:8px">&rarr;</span>'),
-                new DropdownField('TimeoutTransitionID', 'And then perform transition:', $transitions),
-            ])
-        );
+        $groupFields = [
+            DropdownField::create('TimeoutType', 'Timeout Type', $types)
+        ];
+
+        if ($this->TimeoutType == 'Static Period') {
```

Comment:
Given Static Period is used pretty frequently for logic switches, might be worth making it a `const STATIC_PERIOD`